application/values.yaml

```diff
@@ -664,4 +664,36 @@ externalSecret:
       ## Used to fetch all properties from the Provider key
       #   dataFrom:
       #   - key: postgres
-      #     property: credentials
\ No newline at end of file
+      #     property: credentials
+
+##########################################################
+# grafanaDashboard object for defining application 
+# Grafana Dashboard
+##########################################################
+grafanaDashboard:
+  enabled: false
+  
+  # GrafanaDashboard additonal labels
+  additionalLabels:
+    # grafanaDashboard: grafana-operator
+
+  # GrafanaDashboard annotations
+  annotations:
+    # key: value
+  
+  # GrafanaDashboard contents
+  # this includes pairs of dashboard name and associated json content
+  # Accoroding to GrafanaDashboard behavior, if both url and json are specified then the GrafanaDashboard content will be updated with fetched content from url
+  contents:
```

Comment:
@hanzala1234 can one override a particular dashboard or they have to override all of them in next environment?